Libraries/SAModel/ModelFile.cs

```diff
@@ -370,7 +370,7 @@ public static bool CheckModelFile(string filename)
 			}
 		}
 
-		public void SaveToFile(string filename, bool nometa = false, bool useNinjaMetaData = false, bool njbLittleEndian = false)
+		public byte[] ToBytes(string filename, bool nometa = false, bool useNinjaMetaData = false, bool njbLittleEndian = false)

```

Comment:
`GetBytes` would more consistent with existing code.

Reply:
Okay, I'll rename it.